app/models/service_template_ansible_playbook.rb

```diff
@@ -146,13 +148,20 @@ def update_catalog_item(options, auth_user = nil)
       next unless config_info[action]
       info = config_info[action]
 
-      new_dialog = create_new_dialog(info[:new_dialog_name], job_template(action), info[:hosts]) if info[:new_dialog_name]
-      config_info[action][:dialog_id] = new_dialog.id if new_dialog
+      if new_dialog_required(info)
+        new_dialog = create_new_dialog(info[:new_dialog_name], job_template(action), info[:hosts])
+        config_info[action][:dialog_id] = new_dialog.id if new_dialog
+      end
 
-      next unless info.key?(:playbook_id)
-      tower, params = self.class.send(:build_parameter_list, "miq_#{name}_#{action}", description, info)
-      params[:manager_ref] = job_template(action).manager_ref
-      ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript.update_in_provider_queue(tower.id, params, auth_user)
+      if new_job_template_required(info, action)
+        single_config = extract_single_config(info, action)
+        config_info.deep_merge!(self.class.send(:create_job_templates, name, description, single_config, auth_user))
```

Comment:
@bzwei - I believe the change to include `:reconfigure` should be included in a new Feature and subsequent PR(s.) 

This PR is in reference to solving the case mentioned in [#1438839](https://bugzilla.redhat.com/show_bug.cgi?id=1438839) and was built to fix only that problem.